services/dr-manhattan/event_to_kafka.py

```diff
@@ -2,21 +2,23 @@
 from kafka.errors import KafkaError
 from datetime import datetime
 import json
+from operator import itemgetter as iget
 
-def mongo_to_kafka(rec, campaign_thresh = 0.7, debug=False):
+def to_qcr_format(rec, campaign_thresh = 0.7, debug=False):
     if debug:
         print "Start conv, doing location"
         print "rec['location'] = ", rec['location']
 
-    loc = sorted(rec['location'], key=lambda x: x['weight'], reverse=True)
+    loc = sorted(rec['location'], key=iget('weight'), reverse=True)
```

Comment:
i doubt it. i saw this convenience helper when reading up on `sorted` builtin. its just a convenience for the oft-used `x -> x[n]` lambda. i wish we had this option in JS.